.circleci/README.md

```diff
@@ -0,0 +1,81 @@
+PyNE-CI Documentation
+=====================
+
+
+PyNE Circle-CI use CIrcle_CI version 2.1 and is composed in 4 main sections:
+`executors`, `commands`, `jobs` and `workflow`.
+
+Executors
+---------
+The `executors` define aliases to the environment to run the different jobs, in our case
+different docker images containing different combinations of the optional
+dependencies of PyNE (`MOAB`, `pyMOAB`, `DAGMC`) in one of the two supported
+PyTHON flavors (Python 2.7 or Python 3.6).
+
+
+Commands
+---------
+The `commands` section allows the definition of macro commands with arguments to be used
```

Comment:
```suggestion
The `commands` section provides the definitions of macro commands with arguments to be used
```

Reply:
I think this wording is more of what you wanted here? Not sure "allows" is truly what is happening.